tests/rustdoc-gui/implementors.goml

```diff
@@ -2,18 +2,45 @@
 // have the same display than the "local" ones.
 go-to: "file://" + |DOC_PATH| + "/implementors/trait.Whatever.html"
 assert: "#implementors-list"
-// There are supposed to be two implementors listed.
-assert-count: ("#implementors-list .impl", 2)
+// There are supposed to be four implementors listed.
+assert-count: ("#implementors-list .impl", 4)
+// There are supposed to be two non-negative implementors.
+assert-count: ("#implementors-list .negative-marker ~ *", 2)
```

Comment:
```suggestion
// There are supposed to be two negative implementors.
assert-count: ("#implementors-list .negative-marker", 2)
```

This seems much simpler and more reliable, unsure if there's a reason it wasn't done.